src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php

```diff
@@ -84,7 +84,7 @@ class LanguageDataGenerator extends AbstractDataGenerator
         'zh' => 'zho',
     ];
     private static $blacklist = [
-        'root' => true,
+        'root' => true, // Absolute root language
```

Comment:
I'm sorry but this comment doesn't help much. What about something like: `Special non-existent language used for ...`

Reply:
I dont know what it's used for :) `root` is not part of the ISO 639-3 codes.

AFAIK it's only used for grouping; if `en` is a root language, then `root` is the absolute language root.